<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RouteMultiExportDialog</class>
 <widget class="QDialog" name="RouteMultiExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Little Navmap - Select Flight Plan Formats for Multiexport</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>2</number>
     </property>
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <item row="2" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBoxRouteExportOptions">
       <property name="toolTip">
        <string/>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <item>
        <property name="text">
         <string>Show file dialog for every exported format</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Do not show file dialog. Backup existing files in case of conflict</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Do not show file dialog. Overwrite existing files without warning</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelRouteExportOptions">
       <property name="text">
        <string>&amp;Export options:</string>
       </property>
       <property name="buddy">
        <cstring>comboBoxRouteExportOptions</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QTableView" name="tableViewRouteExport">
       <property name="editTriggers">
        <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="cornerButtonEnabled">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>3</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>23</number>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBoxRouteExport">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionSelectExportPath">
   <property name="icon">
    <iconset resource="../../littlenavmap.qrc">
     <normaloff>:/littlenavmap/resources/icons/fileopen.svg</normaloff>:/littlenavmap/resources/icons/fileopen.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Select Export Path ...</string>
   </property>
  </action>
  <action name="actionResetExportPath">
   <property name="icon">
    <iconset resource="../../littlenavmap.qrc">
     <normaloff>:/littlenavmap/resources/icons/reset.svg</normaloff>:/littlenavmap/resources/icons/reset.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Reset Export Path</string>
   </property>
  </action>
  <action name="actionExportFileNow">
   <property name="icon">
    <iconset resource="../../littlenavmap.qrc">
     <normaloff>:/littlenavmap/resources/icons/filesaveas.svg</normaloff>:/littlenavmap/resources/icons/filesaveas.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Export File Now ...</string>
   </property>
  </action>
  <action name="actionResetView">
   <property name="icon">
    <iconset resource="../../littlenavmap.qrc">
     <normaloff>:/littlenavmap/resources/icons/cleartable.svg</normaloff>:/littlenavmap/resources/icons/cleartable.svg</iconset>
   </property>
   <property name="text">
    <string>Reset &amp;View</string>
   </property>
  </action>
  <action name="actionIncreaseTextSize">
   <property name="text">
    <string>&amp;Increase Text Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionDecreaseTextSize">
   <property name="text">
    <string>&amp;Decrease Text Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionEditPath">
   <property name="text">
    <string>&amp;EditPath</string>
   </property>
  </action>
  <action name="actionDefaultTextSize">
   <property name="text">
    <string>Default Text &amp;Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionResetPathsAndSelection">
   <property name="text">
    <string>Reset &amp;Paths and Selection</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tableViewRouteExport</tabstop>
  <tabstop>comboBoxRouteExportOptions</tabstop>
 </tabstops>
 <resources>
  <include location="../../littlenavmap.qrc"/>
 </resources>
 <connections/>
</ui>
